### DISCUSSION TITLE
Unable to Run code Successfully on Visual Studio

### DISCUSSION THREAD

**Discussion by chokostica** [community/community#66381] [New to GitHub]
[csma-cd-protocol-main.zip](https://github.com/community/community/files/12551416/csma-cd-protocol-main.zip)

I cloned the attached repository on Visual Studio C++. It appears there is no .sln file and hence the code does not run. Someone help with steps how I can run this code successfully on C++.

**Comment by queenofcorgis** [comment]
Hi @udlondy , we're happy you're here! You are more likely to get a useful response if you are explicit about what your questions entails, giving a few more details or screenshots might help someone give you a nudge in the right direction as many users may not be inclined to download a zip file (and we don't recommend doing so either).
